test/basic.html

```diff
@@ -30,7 +30,7 @@
 						// Set input value
 						autocomplete.text = value.toString();
 						// Trigger create suggestions
-						suggestions._input.fire('keyup');
+						suggestions.dispatchEvent(new CustomEvent('keyup', { bubbles: true, composed: true }));
```

Comment:
Is there a reason for removing `_input` ?

Reply:
No, this was a mistake that also halted the tests. Added it back.